### PR TITLE
Add invoke actor method for gRPC

### PR DESCRIPTION
**Pull request by yaron2**
Closes https://github.com/dapr/dapr/issues/2400

### CODE REVIEW

File: dapr/proto/runtime/v1/dapr.proto [hunk 1]
```diff
@@ -251,3 +255,16 @@ message UnregisterActorTimerRequest {
...
+message InvokeActorRequest {
+  string actor_type = 1;
+  string actor_id = 2;
+  string method = 3;
+  bytes  data = 4;
+}
+
+// InvokeActorResponse is the method that returns an actor invocation response.
+message InvokeActorResponse {
+  google.protobuf.Any data = 1;
```
Comment (artursouza) [inline dapr/proto/runtime/v1/dapr.proto:269]: Can we use bytes? This is more predictable behavior in different languages for SDKs.

Comment (yaron2) [reply dapr/proto/runtime/v1/dapr.proto]: Changed.

File: pkg/grpc/api.go [hunk 2]
```diff
@@ -611,6 +613,30 @@ func (a *api) UnregisterActorTimer(ctx context.Context, in *runtimev1pb.Unregist
...
+		return &runtimev1pb.InvokeActorResponse{}, err
+	}
+
+	req := invokev1.NewInvokeMethodRequest(fasthttp.MethodPut)
+	req.WithActor(in.ActorType, in.ActorId)
+	req.WithRawData(in.Data, "")
+
+	resp, err := a.actor.Call(context.TODO(), req)
+	if err != nil {
+		err := status.Errorf(codes.Internal, messages.ErrActorInvoke, err)
```
Comment (artursouza) [inline pkg/grpc/api.go:629]: lint here.

Comment (yaron2) [reply pkg/grpc/api.go]: Resolved

**Review by artursouza — APPROVED** [review]

**Pull request merged**